pkgs/by-name/cs/csvtk/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  ...
+}:
```

Comment:
```suggestion
}:
```

Reply:
Could you update this so I can approve? I've seen other PRs where it is also asked, and just a handful of packages has it like this.

Reply:
@smancill  Done!